### ISSUE TITLE
restore data from a mongoDB container

### ISSUE THREAD

**Issue by thsurgeonofdeath** [RocketChat/Docker.Official.Image#188]
Hello, I have a rocket chat server (v 5.2.0) running on docker container and then backed up data using: docker exec db sh -c "mongodump --archive" > db.dump
I created a docker-compose file, that runs a new instance of rocket chat and MongoDB, then restored the dump file using: docker exec -i db sh -c "mongorestore --archive" < db.dump. 
After the docker-compose startup, even when the logs show data transfer was successful, I'm always redirected to the default setup wizard, I looked into it and found i needed to add `OVERWRITE_SETTING_Show_Setup_Wizard=completed` in the environment, but even then, no data is displayed, it just asks me to log in, the credentials were wrong and I had to create a new user, and even then it's the default rocket chat interface with none of my rooms, channels, custom css...
How can add the data from a dumpfile of a previous rocketchat server to this new one running using docker-compose?

Thanks

**Comment by mremptypants** [comment]
Yeah, I am getting the same thing, mine started with the upgrade so i rebuilt everything moved the mongodb, 7.10 did the import and wizrd and non user found, it works if the users never log out...
